### ISSUE TITLE
No foreign key created

### ISSUE THREAD

**Issue by mickeywu** [balderdashy/sails-postgresql#123]
When a relationship is defined between 2 models, no foreign key constraint gets created in the actual PostgreSQL DB.

Here's an example relationship:

*Company.js*
```javascript
module.exports = {

  attributes: {
  	name: {
      type: 'string',
      required: true
    },
    monitors: {
      collection: 'monitor',
      via: 'company'
    }
  }

};
```

*Monitor.js*
```javascript
module.exports = {

  attributes: {
    name: {
      type: 'string',
      required: true
    },
    company: {
      model: 'company',
      foreignKey: true
    }
  }

};
```

Here's the Monitor table in PostgreSQL:

```
                                   Table "public.monitor"
  Column   |           Type           |                      Modifiers                       
-----------+--------------------------+------------------------------------------------------
 name      | text                     | 
 company   | integer                  | 
 id        | integer                  | not null default nextval('monitor_id_seq'::regclass)
 createdAt | timestamp with time zone | 
 updatedAt | timestamp with time zone | 
Indexes:
    "monitor_pkey" PRIMARY KEY, btree (id)
```

As you can see, there's no foreign key constraint created.  If I do the same thing with MySQL instead, it will create a FK constraint.

Is there something that I'm missing or is this the intended behaviour?

Thanks in advance.

**Comment by miguelcobain** [comment]
Also looking for this.

**Comment by ex37** [comment]
+1, no way to create foreign keys

**Comment by Zigur** [comment]
+1, I have found the same issue

**Comment by eriklindgren** [comment]
Agreed, having the same issue.

**Comment by bastoune** [comment]
I am also having this issue !

**Comment by sailsbot** [comment]
Thanks for posting, @mickeywu.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by mrtensai** [comment]
Any solution for this issue yet ?

**Comment by ManeRs11** [comment]
Any solution ?

**Comment by eriklindgren** [comment]
We decided to work around the issue by disabling Sails migrations and implementing our own migrations using [Knex](http://knexjs.org/#Migrations).

**Comment by feardarkness** [comment]
Any solution?
